boray/src/bucket/create.rs

```diff
@@ -60,8 +60,10 @@ pub(crate) fn action(
             // Database errors are returned to as regular Fast messages
             // to be handled by the calling application
             let value = array_wrap(json!({
-                "name": "PostgresError",
-                "message": e
+                "error": {
```

Comment:
One suggestion that could cut down on some of the repetition might be to add a `PostgresError` type to the `BorayErrorType` in the `error` module. Then if add a function similar to [this](https://github.com/joyent/buckets-mdapi/blob/master/boray/src/bucket.rs#L58-L63) one for the `BucketAlreadyExists` variant we can consolidate these 4 lines in to 1 everywhere the `PostgresError` happens and the shape of the JSON response is determined in one place versus many.

Reply:
Will make this change